### DISCUSSION TITLE
"Free Motion Capture for Everyone" and you need to be a coder to launch the thing....

### DISCUSSION THREAD

**Discussion by dennypool** [freemocap/freemocap#261] [General]
i mean, this project have more than 2 years? maybe you can build a ui or something so even coder noobs can use the project??? i mean 95% of people or even more can't write a single line of code and for sure can't launch this awesome project, "for everyone" doesn't sound very right

**Comment by aaroncherian** [comment]
Thanks for your interest in FreeMoCap! As you may have noted on our home page, we're currently in the process of transitioning from a pre-alpha version into an alpha version that does have a GUI. While the alpha version still requires some coding knowledge to get started, making FreeMoCap something that you can launch without having to write a single line of code is absolutely our ultimate goal, but those things just take time to do correctly :) If that's what you're looking for, be sure to keep checking back in for our progress and updates!
